tests/executors/test_celery_executor.py

```diff
@@ -75,6 +75,17 @@ def _prepare_app(broker_url=None, execute=None):
     patch_app = mock.patch('airflow.executors.celery_executor.app', test_app)
     patch_execute = mock.patch('airflow.executors.celery_executor.execute_command', test_execute)
 
+    backend = test_app.backend
```

Comment:
Does this issue only occur for tests and not likely to occur in the actual code / Airflow deployment?

Reply:
This is unlikely to happen in a real environment, because there is a very small chance that all processes/deployment will synchronize. The database is also not created many times in a production environment.